ddspipe_core/include/ddspipe_core/configuration/DdsPipeLogConfiguration.hpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <cpp_utils/Formatter.hpp>
+#include <cpp_utils/logging/CustomStdLogConsumer.hpp>
+
+#include <ddspipe_core/configuration/DdsPublishingConfiguration.hpp>
+#include <ddspipe_core/library/library_dll.h>
+
+namespace eprosima {
+namespace ddspipe {
+namespace core {
+
+/**
+ * TODO
+ */
+struct DdsPipeLogConfiguration : public utils::LogConfiguration
+{
+
+    /////////////////////////
+    // CONSTRUCTORS
+    /////////////////////////
+
+    DDSPIPE_CORE_DllAPI
+    DdsPipeLogConfiguration() = default;
+
+    /////////////////////////
+    // METHODS
+    /////////////////////////
+
+    DDSPIPE_CORE_DllAPI
+    virtual bool is_valid(
+            utils::Formatter& error_msg) const noexcept override;
+
+    /////////////////////////
+    // VARIABLES
+    /////////////////////////
+
+    DdsPublishingConfiguration publish;
+
+    bool stdout_enable = false;
```

Comment:
If we don't change stdout in YAML but set --log-verbosity info in the command-line, will the logs still not be displayed? This implies that if you wish to configure them via the command-line, you'll also need to set stdout=true in YAML. I notice a slight inconsistency in having to configure them in both places. Perhaps this variable could default to true if the log and/or filter are configured via the command-line?